src/oc_erchef/apps/migrator/test/migrator_tests.erl

```diff
@@ -0,0 +1,29 @@
+-module(migrator_tests).
+
+-compile([export_all]).
```

Comment:
this `export_all` can be an issue if you plan to support OTP 20 in the future, even on 19 it may complain, can use:

```
-compile([nowarn_export_all, export_all]).
```